experiment/src/cortex/experiment/util.clj

```diff
@@ -6,6 +6,39 @@
   (:import [java.io File]))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Data encoding utils
+
+(defn label->one-hot
+  "Given a vector of class-names and a label, return a one-hot vector based on
+  the position in class-names.
+  E.g.  (label->vec [:a :b :c :d] :b) => [0 1 0 0]"
+  [class-names label]
+  (let [num-classes (count class-names)
```

Comment:
[mild style thought] If a local appears only once in the code that follows its binding, I always consider eliminating it. In this case I think it would be fine to elide this and just move `(count ...)` into the next expression.